ansible/vars/dev.yml

```diff
@@ -47,6 +47,12 @@ postgres_users:
     username: "{{ localsettings.PG_DATABASE_USER }}"
     password: "{{ localsettings.PG_DATABASE_PASSWORD }}"
 
+postgres_config:
```

Comment:
Isn't this a huge decrease in max_connections? Did you use a different number on prod?